socorro/app/for_application_defaults.py

```diff
@@ -43,8 +43,7 @@ def find_all_the_apps():
         return {
             'setupdb': 'socorro.external.postgresql.setupdb_app.SocorroDBApp',
             'submitter': 'socorro.submitter.submitter_app.SubmitterApp',
-            # crontabber not yet supported in this environment
-            #'crontabber': 'socorro.cron.crontabber_app.CronTabberApp',
+            'crontabber': 'socorro.cron.crontabber_app.CronTabberApp',
```

Comment:
This appears to be a project that was started a few years ago:

https://github.com/mozilla-services/socorro/commit/3f469e41d600e50f706e62a027e235bd16420eca

but it wasn't finished for crontabber. This PR fixes that for crontabber.